documentation/src/developer_documentation/state_migration_spike.md

```diff
@@ -3,7 +3,8 @@
 ### What is state migration?
 
 State migration is a process where the `StateTree` contents are transformed from
-an older form to a newer form. Certain Actors may need to be created or migrated as well.
```

Comment:
Changed in this file is unrelated but it fixes the CI failure https://github.com/ChainSafe/forest/actions/runs/4562495209/jobs/8050243844#step:3:15